File: Domain/Entry.cs
using System;

namespace QuestLog.Domain
{
    public class Entry : DbBase
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime Date { get; set; }
        public DateTime? EndDate { get; set; }
        public bool Completed { get; set; }

        public Guid UserId { get; set; }
    }
}

File: Client/src/state/entry.js
export const InitialState =  {
    id: null,
    title: '',
    description: '',
    date: new Date(),
    endDate: null,
    completed: false,
    displayArea: 0,
    userId: ''
};

export const DisplayAreas = {
    Day: {id: 0, name: '[[Replace Me]]'},
    Todo: {id: 1, name: 'To Do'},
    Note: {id: 2, name: 'Notes'}
}